judge/handler.py

```diff
@@ -87,15 +106,6 @@ def process_problem(code: str, contest: int, name: str, statement: str, input_fo
     except models.Problem.DoesNotExist:
         pass
 
-    # Set up default values
-    cp_comp_script, cp_test_script = False, False
```

Comment:
compilation_script and test_script are non-nullable in db. If you do not change None to sth else here, db will complain